python_modules/libraries/dagster-components/dagster_components/lib/dbt_project/component.py

```diff
@@ -26,50 +26,41 @@
 from dagster_components.utils import TranslatorResolvingInfo, get_wrapped_translator_class
 
 
-class ResolvedDbtProjectSchema(BaseModel):
-    dbt: DbtCliResource
-    op: OpSpecSchema
-    translator: DagsterDbtTranslator
-    transforms: Sequence[Callable[[Definitions], Definitions]]
-
-
-class DbtProjectSchema(ResolvableSchema[ResolvedDbtProjectSchema]):
+class DbtProjectSchema(ResolvableSchema["DbtProjectComponent"]):
     dbt: DbtCliResource
```

Comment:
note: it would be really helpful I think if you got rid of all these resources that are directly on scheam. It really does pollute the mental model IMO